File: src/main/java/com/uwetrottmann/trakt5/enums/Type.java
package com.uwetrottmann.trakt5.enums;

public enum Type implements TraktEnum {

    MOVIE("movie"),
    SHOW("show"),
    EPISODE("episode"),
    PERSON("person"),
    LIST("list");

    private final String value;

    private Type(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return value;
    }

}

File: src/main/kotlin/com/redissi/trakt/enums/Type.kt
package com.redissi.trakt.enums

enum class Type(private val value: String) : TraktEnum {
    MOVIE("movie"),
    SHOW("show"),
    EPISODE("episode"),
    PERSON("person"),
    LIST("list");

    override fun toString(): String {
        return value
    }

}